_schemas/1.0/tests/response/invalid/resource/relationship_named_id.json

```diff
@@ -0,0 +1,17 @@
+{
+  "data": {
+    "type": "article",
+    "id": "1",
+    "attributes": {
+      "title": "Rails is Omakase"
```

Comment:
I would recommend to use a neutral example here as well. This one may not be as offending as the PHP example above. But I think for a specification it would be to not make any opinionated statements about technologies, which might be used to implement it.

Reply:
Same as for PHP...

Reply:
As I said for PHP, it's a copy/paste of an exemple found in the specs : [https://jsonapi.org/format/#document-resource-objects](https://jsonapi.org/format/#document-resource-objects).

Reply:
Done